commit 44d313f9c06dfe573d479b78dd361def419aefa4 (refs/changes/56/4456/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-06-29T01:46:20+00:00 (1 year, 3 months ago)
    
    joyent/node-zkstream#34 use new cueball decoherence timeout to avoid rapid shuffling
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/client.js b/lib/client.js
index bb76ef7..4113b8e 100644
--- a/lib/client.js
+++ b/lib/client.js
@@ -107,6 +107,8 @@ function ZKClient(opts) {
 		},
 		target: 1,
 		maximum: 3,
+		/* ZK connections can be expensive, don't shuffle too much. */
+		decoherenceInterval: 600,
 		connectionHandlesError: true,
 		constructor: this._makeConnection.bind(this)
 	});
diff --git a/package.json b/package.json
index 2b8146c..cff100b 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "zkstream",
-  "version": "0.11.1",
+  "version": "0.11.2",
   "description": "",
   "repository": {
     "type": "git",
@@ -11,7 +11,7 @@
     "artedi": "1.3.0",
     "assert-plus": "^1.0.0",
     "bunyan": "^1.8.10",
-    "cueball": "^2.2.7",
+    "cueball": "^2.7.0",
     "jsbn": "^0.1.0",
     "mooremachine": "^2.0.0",
     "node-uuid": "^1.4.7",
